### PR TITLE
refactor: layouts use setup

### PR DESCRIPTION
**Pull request by xingyuv**
### `General`

> ✏️ Mark the necessary items without changing the structure of the PR template.

- [ ] Pull request template structure not broken

### `Type`

> ℹ️ What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### `Checklist`

> ℹ️ Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](contributing.md).

> 👉 _Put an `x` in the boxes that apply._

- [ ] My code follows the style guidelines of this project
- [ ] Is the code format correct
- [ ] Is the git submission information standard?
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by wangjue666** [comment]
@xingyuv  看下 `ci `信息 有ts类型问题

**Comment by wangjue666** [comment]
先弄成草稿，完善了再重新打开

File: src/layouts/default/header/components/Breadcrumb.vue [hunk 1]
```diff
@@ -1,153 +1,152 @@
...
+        return false;
       }
+      return true;
+    }).filter((item) => !item.meta?.hideBreadcrumb);
+  }
 
-      function handleClick(route: RouteLocationMatched, paths: string[], e: Event) {
-        e?.preventDefault();
-        const { children, redirect, meta } = route;
+  function handleClick(route: any, paths: string[], e: Event) {
```
Comment (wangjue666) [inline src/layouts/default/header/components/Breadcrumb.vue:111]: 为啥把类型变成any了

Comment (xingyuv) [reply src/layouts/default/header/components/Breadcrumb.vue:111]: Breadcrumb 的 route 类型与系统内的不一致

Comment (xingyuv) [reply src/layouts/default/header/components/Breadcrumb.vue:111]: interface Route {
  path: string;
  breadcrumbName: string;
  children?: Array<{
    path: string;
    breadcrumbName: string;
  }>;
}

这是 breadcrumb 的类型定义

**Review by wangjue666 — APPROVED** [review]

File: src/layouts/default/header/components/Breadcrumb.vue (hunk 1 above)
Comment (wangjue666) [reply src/layouts/default/header/components/Breadcrumb.vue]: 那这个文件先不要动它好了

**Pull request merged**